### PR TITLE
added 0.9.3.0 version of kafka-mesos

### PR DESCRIPTION
**Pull request by dmitrypekar**
Hi.

I've added 0.9.3.0 version of kafka-mesos to the repo.
Please review & merge.

### CODE REVIEW

**Comment by brndnmtthws** [comment]
I'll need to update #267 as well, if this gets merged first.

**Comment by kensipe** [comment]
wish I reviewed this before I merged.  reviewing now.

**Comment by kensipe** [comment]
@brndnmtthws could we get the `heap-mb` fix to this as well?

File: repo/packages/K/kafka/2/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "Apache Kafka running on top of Apache Mesos",
+  "framework": true,
+  "tags": ["message", "broker", "messaging"],
+  "images": {
+    "icon-small": "https://d1vubr0evspla.cloudfront.net/img/icon-small.png",
+    "icon-medium": "https://d1vubr0evspla.cloudfront.net/img/icon-medium.png",
+    "icon-large": "https://d1vubr0evspla.cloudfront.net/img/icon-large.png",
+    "screenshots": ["https://d1vubr0evspla.cloudfront.net/img/screen-0-kafka.png"]
+  },
+  "preInstallNotes": "This will install Apache Kafka DCOS Service.",
```
Comment (kensipe) [inline repo/packages/K/kafka/2/package.json]: The purpose of the pre install notes is to provide the user with information to make an install decision.  We typically have resource requirements here... like number of nodes, cpu and mem requirements.

File: repo/packages/K/kafka/2/config.json [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    "type": "string",
+                    "description": "Other Scheduler options if required",
+                    "default": ""
+                }
+            },
+            "required": [
+                "app",
+                "zk", "api", "storage", "jre",
+                "framework-name", "framework-role",
+                "user", "principal", "secret",
```
Comment (kensipe) [inline repo/packages/K/kafka/2/config.json]: required means that it is used by the marathon json in a way that isn't conditional.  If there is a condition check then it shouldn't be listed as required.  So... user, principal, secret are NOT required.

**Comment by brndnmtthws** [comment]
This is actually not my PR, so I can't update it.

But yes, I agree with you on the notes above.

**Comment by kensipe** [comment]
@dmitrypekar let me know when you might have an update

**Comment by kensipe** [comment]
@dmitrypekar also please look at closed PR 267 for need Java mx updates.

**Comment by gabrielhartmann** [comment]
@dmitrypekar:  Apologies for the delay on handling this pull request.  Can you please make this a PR against version-2.x?

**Comment by gabrielhartmann** [comment]
@dmitrypekar:  I'm going to close this for now.  We probably want a new request against version-2.x anyway.  Let me know if there are any problems with this.

**Pull request closed without merging**